CRM/Event/BAO/Query.php

```diff
@@ -90,6 +90,13 @@ public static function select(&$query) {
         $query->_element['participant_fee_amount'] = 1;
       }
 
+      if (!empty($query->_returnProperties['participant_contribution_id'])) {
+        $query->_select['participant_contribution_id'] = "civicrm_participant_payment.participant_id as participant_contribution_id";
```

Comment:
I think this is an unhelpful renaming - please change to participant_payment_id.